apps/web/src/components/Composer/Post/New.tsx

```diff
@@ -89,7 +89,7 @@ const NewPost: FC = () => {
           </span>
         </button>
         <Modal
-          title={t`Create post`}
+          title={t`CREATE POST`}
```

Comment:
Changing the translation key breaks the translation.
Maybe the easiest is to add an `uppercase` class to the Modal's title itself?